### PR TITLE
Consumer API: Rename AcceptanceContent property of Relationship class

### PR DESCRIPTION
**Pull request by NikolaVetnic**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

# Description

Rename the `AcceptanceContent` property of `Relationship` class to `CreationResponseContent`. Enable sending content when rejecting and revoking a Relationship.

### CODE REVIEW

File: Modules/Relationships/src/Relationships.Application/Relationships/DTOs/RelationshipMetadataDTO.cs [hunk 1]
```diff
@@ -25,6 +26,7 @@ public RelationshipMetadataDTO(Relationship relationship)
 
     public DateTime CreatedAt { get; set; }
 
+    public byte[]? CreationResponseContent { get; set; }
```
Comment (tnotheis) [inline Modules/Relationships/src/Relationships.Application/Relationships/DTOs/RelationshipMetadataDTO.cs:29]: Why did you add this? The content hasn't been there before intentionally. This is the Relationship**Metadata**DTO, which means it doesn't have content. It is used for responses where we don't care about the content, namely the responses of all actions.

**Review by tnotheis — CHANGES_REQUESTED** [review]

File: Modules/Relationships/src/Relationships.Application/Relationships/DTOs/RelationshipMetadataDTO.cs (hunk 1 above)
Comment (NikolaVetnic) [reply Modules/Relationships/src/Relationships.Application/Relationships/DTOs/RelationshipMetadataDTO.cs]: My bad, I misunderstood the requirements then.

Changes made.

**Review by tnotheis — APPROVED** [review]
